api/src/lib/token.js

```diff
@@ -0,0 +1,23 @@
+const jsonWebToken = require('jsonwebtoken')
+
+module.exports = class Token {
+  constructor (deps) {
+    this.config = deps(Config)
+    this.secret = this.config.data.getIn(['api', 'secret'])
+  }
+
+  get (prefix, duration) {
+    return jsonWebToken.sign({
+      prefix: prefix,
+      exp: String(Date.now() + duration)
```

Comment:
For duration, you pass in `Infintiy`. This is not a valid value for `exp`: `ValidationError: child "exp" fails because ["exp" contains an invalid value]`